lib/node_modules/@stdlib/math/base/complex/divide/benchmark/cdiv-smith.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// MAIN //
+
+/**
+* Divides two complex numbers.
+*
+* @param {number} re1 - real component
+* @param {number} im1 - imaginary component
+* @param {number} re2 - real component
+* @param {number} im2 - imaginary component
+* @returns {Array<number>} real and imaginary components
+*
+* @example
+* var v = cdiv( 5.0, 3.0, -2.0, 1.0 );
+* var v = cdiv( -13.0, -1.0, -2.0, 1.0 );
+* // returns [ 5.0, 3.0 ]
+*/
+function cdiv( re1, im1, re2, im2 ) {
```

Comment:
I left this version of the function in place in case I should extend the benchmarks to compare Smith's algorithm (or the naive algorithm too, maybe).